### PR TITLE
Full linting with flake8

### PR DESCRIPTION
**Pull request by dataflake**
This PR integrates code linting with flake8 into the tox configuration and implements it throughout the code.

The code changes are 99.9% cosmetic.

### CODE REVIEW

File: src/ZEO/asyncio/tests.py [hunk 1]
```diff
@@ -2,17 +2,17 @@
...
     import asyncio
+
     def to_byte(i):
         return bytes([i])
 else:
-    import trollius as asyncio
+    import trollius as asyncio  # NOQA: F401 unused import
+
     def to_byte(b):
         return b
```
Comment (d-maurer) [inline src/ZEO/asyncio/tests.py:12]:
```suggestion
from .compat import asyncio

if PY3:
    def to_byte(i):
        return bytes( [i])
else:
    def to_byte(b):
        return b
```

File: src/ZEO/scripts/cache_simul.py [hunk 2]
```diff
@@ -433,7 +434,8 @@ def write(self, oid, size, start_tid, end_tid, evhit=0):
             # Storing current revision.
             if oid in self.current:  # we already have it in cache
                 if evhit:
-                    import pdb; pdb.set_trace()
+                    import pdb
+                    pdb.set_trace()
```
Comment (d-maurer) [inline src/ZEO/scripts/cache_simul.py:438]: Do we really want debugging code here?

File: src/ZEO/scripts/cache_simul.py [hunk 3]
```diff
@@ -442,7 +444,8 @@ def write(self, oid, size, start_tid, end_tid, evhit=0):
             self.add(oid, size, start_tid)
             return
         if evhit:
-            import pdb; pdb.set_trace()
+            import pdb
+            pdb.set_trace()
```
Comment (d-maurer) [inline src/ZEO/scripts/cache_simul.py:448]: Debugging code?

**Review by d-maurer — APPROVED** [review]
Have you made the changes manually or have you used a tool? I ask because I will need to redo part of this in the `asyncio3.5+` branch (sooner or later).

File: src/ZEO/asyncio/tests.py (hunk 1 above)
Comment (dataflake) [reply src/ZEO/asyncio/tests.py:12]: master still supports Python 2.7, this must stay.

**Comment by dataflake** [comment]
> Have you made the changes manually or have you used a tool? I ask because I will need to redo part of this in the `asyncio3.5+` branch (sooner or later).

I did all of it manually and since that was very tedious I did not spend any time thinking about e.g. old debugging code. That wasn't the goal of this exercise.

**Comment by d-maurer** [comment]
Jens Vagelpohl wrote at 2022-4-7 05:42 -0700:
***@***.*** commented on this pull request.
>>  if PY3:
>     import asyncio
>+
>     def to_byte(i):
>         return bytes([i])
> else:
>-    import trollius as asyncio
>+    import trollius as asyncio  # NOQA: F401 unused import
>+
>     def to_byte(b):
>         return b
>
>master still supports Python 2.7, this must stay.

I suggested to import `asyncio` from `.compat` (where the `trollius/asyncio`
switch is alreasy implemented) not to drop it.

**Comment by dataflake** [comment]
You're right, I didn't read that closely enough. I will push a fix right to master once the tox tests have run, should be another 5-10 minutes.

**Comment by d-maurer** [comment]
Jens Vagelpohl wrote at 2022-4-7 08:49 -0700:
>You're right, I didn't read that closely enough. I will push a fix right to master once the tox tests have run, should be another 5-10 minutes.

It is just cosmetics -- perhaps not worth a special action.

**Pull request merged**